File: restdocs-example/src/main/java/io/github/ivvve/restdocs/demo/EchoController.java
package io.github.ivvve.restdocs.demo;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class EchoController {
    @GetMapping("/echo")
    public String echo(@RequestParam final String value) {
        return value;
    }
}

File: kotlin-gradle-dsl-restdocs-error/src/main/kotlin/io/github/ivvve/restdocs/EchoController.kt
package io.github.ivvve.restdocs

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController

@RestController
class EchoController {
    @GetMapping("/echo")
    fun echo(@RequestParam value: String): String {
        return value
    }
}